### PR TITLE
Thoughtbot should use thoughtbot config

### PR DESCRIPTION
**Pull request by jcmorrow**
Migrate thoughtbot owner to use the guides as its owner level config.

### CODE REVIEW

File: db/migrate/20160624165658_update_thoughtbot_owner_configs.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class UpdateThoughtbotOwnerConfigs < ActiveRecord::Migration
+  def up
+    execute <<~SQL
+      UPDATE
+        owners
+      SET
+        config_repo = 'thoughtbot/guides'
+      WHERE
+        owners.name = 'thoughtbot'
```
Comment (thorncp) [inline db/migrate/20160624165658_update_thoughtbot_owner_configs.rb]: `owners.name in ('thoughtbot', 'houndci')` ?

Comment (gylaz) [inline db/migrate/20160624165658_update_thoughtbot_owner_configs.rb]: Good call on `houndci`. We could test it right away.

**Comment by thorncp** [comment]
LGTM :bank:

File: db/migrate/20160624165658_update_thoughtbot_owner_configs.rb (hunk 1 above)
Comment (jcmorrow) [inline db/migrate/20160624165658_update_thoughtbot_owner_configs.rb]: Ah yeah, good call.

**Pull request merged**